### PR TITLE
Remove AtomicUsize deprecations 

### PR DESCRIPTION
**Pull request by flouthoc**
* Remove trailing colon , make build happy.
* Remove deprecated `compare_and_swap`
* Bump package to current release.

### CODE REVIEW

File: src/devices/src/virtio/fs/linux/passthrough.rs [hunk 1]
```diff
@@ -665,8 +665,8 @@ fn forget_one(
             // Synchronizes with the acquire load in `do_lookup`.
             if data
                 .refcount
-                .compare_and_swap(refcount, new_count, Ordering::Release)
-                == refcount
+                .compare_exchange(refcount, new_count, Ordering::Release, Ordering::Release)
```
Comment (slp) [inline src/devices/src/virtio/fs/linux/passthrough.rs:668]: According to [this](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicUsize.html#method.compare_and_swap), the fourth argument should be `Ordering::Relaxed`.

**Review by slp — CHANGES_REQUESTED** [review]
Thanks for the PR! A couple of comments:

- This should probably be two different commits, one removing the unneeded semicolons, and the other replacing `compare_and_swap` with `compare_exchange`.
- The commit addressing the replacement for `compare_and_swap` should also replace the occurrence at `src/logger/src/logger.rs:318`.
- Please add a [S-o-B](https://www.kernel.org/doc/html/v4.17/process/submitting-patches.html#developer-s-certificate-of-origin-1-1) to each commit, and a brief commit description in addition to the title.

**Comment by flouthoc** [comment]
@slp added all the changes and created separate/signed commit for every change.

File: src/devices/src/virtio/fs/linux/passthrough.rs (hunk 1 above)
Comment (flouthoc) [reply src/devices/src/virtio/fs/linux/passthrough.rs]: Resolved in latest commit.

**Comment by flouthoc** [comment]
ah rustfmt is missing let me do that.

**Comment by flouthoc** [comment]
@slp PTAL

**Comment by flouthoc** [comment]
@slp did fmt and added all the changes with separate/signed commit for every change.

**Review by slp — APPROVED** [review]
LGTM, thanks a lot!

**Pull request merged**